package.json

```diff
@@ -19,6 +20,12 @@
     "build-storybook": "build-storybook"
   },
   "jest": {
+    "collectCoverageFrom": [
+      "**/src/**/*.{js}",
```

Comment:
this covers packages/foo/src?

Reply:
It collects from src based on the tests.  `**` is an N directories before `/src` so yes it is meant to cover everything in packages. We could add in each repo instead if that would be easier.